access/src/main/webapp/WEB-INF/jsp/fullRecord/adminUnit.jsp

```diff
@@ -45,7 +45,21 @@
     <div id="is-admin-unit" class="columns browse-header">
         <div class="column is-12">
             <c:import url="fullRecord/navigationBar.jsp" />
-            <h2><i class="fa fa-university" aria-hidden="true"></i> <c:out value="${briefObject.title}" /></h2>
+            	<c:choose>
```

Comment:
can't you use the `isDeleted` variable here? I don't think having any empty `class` attribute is against any rules